gtsfm/densify/mvs_patchmatchnet.py

```diff
@@ -302,4 +308,4 @@ def filter_depth(
                 joint_mask.mean(),
             )
 
-        return np.concatenate(vertices, axis=0)
+        return {"points": np.concatenate(vertices, axis=0), "rgb": np.concatenate(vertex_colors, axis=0)}
```

Comment:
@ayushbaid @akshay-krishnan could you all take a look at this line?

I think we might be "using a dictionary as a class", which can lead to a bunch of issues.

Would everyone be ok with defining a new class
```python
class MvsResult(NamedTuple)
       points: np.ndarray
       rgb: np.ndarray
```

That we return instead?

Reply:
Whats are the pros/cons of storing the color along with the coordinates vs storing storing them separate as you have done here?